docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+
+### For schedules and sensors
+
+Keep all the policies for triggering jobs together, i.e., put most schedules and sensors together.
+
+In this example, we put sensors and schedules in [`jobs.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/jobs.py). When we build sensors or schedules from a job, they are considered as policies for when to trigger a particular job. So, keeping all the policies together with jobs makes it clear when and how certain jobs are being launched.
+
+- Note: Certain sensors like [run status sensors](/concepts/partitions-schedules-sensors/sensors#run-status-sensors) can listen to multiple jobs and do not trigger a job. We recommend keeping these sensors in the repository definition as they are often for alerting and monitoring at the repository level.
+
+### For resources
+
+Make resources reusable and share them across jobs or asset groups.
+
+In this example, resources (e.g., database connections, Spark sessions, API clients, and I/O managers) are grouped in the [`resources/__init__.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/__init__.py) file, where they are bound to configuration sets which vary based on environment. In complex projects, we find it helpful to make resources reusable and configured with pre-defined values via <PyObject object="configured" />. So when developing a project, instead of trying to find their own way to model external components, teammates can all quickly get stuff done using a pre-defined resource set, or make changes to resources that are shared among the team.
+
+This pattern also helps you to easily execute jobs in different environments without code changes. In this example, we dynamically define a repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+### For ops and graphs
+
+
+Although assets are useful in many ways, [ops](/concepts/ops-jobs-graphs/ops) and [graphs](/concepts/ops-jobs-graphs/graphs), are part of the Dagster ecosystem and can be integrated into your project structure.
```

Comment:
🤔 I feel like the positioning in this section is a little off. The purpose of this guide is to provide **structure** recommendations, not discuss assets vs ops and when to use them. I think we can scrap the content here that does that and instead focus on the recommendations. Ex:

> If using ops, we recommend keeping them together in an `ops_graphs/` directory. Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.

It might also be worthwhile to note that the example project doesn't include this directory, or folks might go looking for it.